src/main/java/hudson/plugins/performance/PerformancePublisher.java

```diff
@@ -147,7 +180,12 @@ public PerformancePublisher(int errorFailedThreshold,
                             boolean modeOfThreshold,
                             boolean compareBuildPrevious,
                             List<? extends PerformanceReportParser> parsers,
-                            boolean modeThroughput) {
+                            boolean modeThroughput,
+                            boolean modeEvaluation,
+                            List<? extends AbstractConstraint> constraints,
+                            boolean ignoreFailedBuilds,
+                            boolean ignoreUnstableBuilds,
+                            boolean persistConstraintLog) {
```

Comment:
Okay, I reviewed this issue again and it feels like I'm kind of stuck here.

1.Approach: Mark old constructor as deprecated and overload it with new constructor.
Problem: see comment above. Found this: https://issues.jenkins-ci.org/browse/JENKINS-11739
So, IMO this approach is not an suitable option.

2.Approach: Use @DataBoundSetter for new parameters
Problem: @DataBoundSetter was introduced at Jenkins version 1.535. This plugin is based on Jenkins 1.480. Thus, we would have to migrate the plugin to a newer Jenkins version.

Any ideas how to handle this issue?